From 8d006eeb4307273108b0418c26914e15c5f7a8bf Mon Sep 17 00:00:00 2001
From: chuangcheng peng <chuangcheng.peng@amlogic.com>
Date: Tue, 25 Apr 2023 18:07:12 +0800
Subject: [PATCH] demux: cut code for S1A chip [1/1]

PD#SWPL-120888

Problem:
cut code for S1A chip

Solution:
cut code for S1A chip

Verify:
not verified

Change-Id: I9991907448b4d4a011d2b2cccc121f4ea68ee82e
Signed-off-by: chuangcheng peng <chuangcheng.peng@amlogic.com>
---
 drivers/media/dvb-core/Kconfig  | 2 +-
 drivers/media/dvb-core/Makefile | 9 +++++++--
 2 files changed, 8 insertions(+), 3 deletions(-)

diff --git a/drivers/media/dvb-core/Kconfig b/drivers/media/dvb-core/Kconfig
index 6ffac618417b..5288d33a55e5 100644
--- a/drivers/media/dvb-core/Kconfig
+++ b/drivers/media/dvb-core/Kconfig
@@ -18,7 +18,7 @@ config DVB_MMAP
 
 config DVB_NET
 	bool "DVB Network Support"
-	default (NET && INET)
+	default (NET && INET) if !AMLOGIC_ZAPPER_CUT
 	depends on NET && INET && DVB_CORE
 	help
 	  This option enables DVB Network Support which is a part of the DVB
diff --git a/drivers/media/dvb-core/Makefile b/drivers/media/dvb-core/Makefile
index 62b028ded9f7..d4c27ac3a8f7 100644
--- a/drivers/media/dvb-core/Makefile
+++ b/drivers/media/dvb-core/Makefile
@@ -6,8 +6,13 @@
 dvb-net-$(CONFIG_DVB_NET) := dvb_net.o
 dvb-vb2-$(CONFIG_DVB_MMAP) := dvb_vb2.o
 
-dvb-core-objs := dvbdev.o dmxdev.o dvb_demux.o			\
+ifndef AMLOGIC_ZAPPER_CUT
+dvb-core-objs := dvbdev.o dmxdev.o dvb_demux.o	\
 		 dvb_ca_en50221.o dvb_frontend.o		\
 		 $(dvb-net-y) dvb_ringbuffer.o $(dvb-vb2-y) dvb_math.o
-
+else
+dvb-core-objs := dvbdev.o dmxdev.o 	\
+		 dvb_ca_en50221.o dvb_frontend.o		\
+		 $(dvb-net-y) dvb_ringbuffer.o $(dvb-vb2-y) dvb_math.o
+endif
 obj-$(CONFIG_DVB_CORE) += dvb-core.o
-- 
2.32.0

